README.md

```diff
@@ -1,2 +1,137 @@
-# School_Library_Manager
-This is a School library management system
+# ENUMERABLE
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [ENUMERABLE] <a name="about-project"></a>
+
+**[ENUMERABLE]** The Enumerable mixin provides collection classes with several traversal and searching methods, and with the ability to sort. The class must provide a method each, which yields successive members of the collection.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">RUBY</a></li>
+  </ul>
+
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Test the same funcionality as methods in Enumerable]**
+
+### Live Demo <a name="live-demo"><a/>
+
+[Live Demo Link]()
+
+### Video <a name="video"><a/>
+
+[Video Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Ruby Installed on System.
+
+Terminal
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd [your-folder] https://github.com/Mukumbuta/Enumerable.git
+cd my-folder
+
+Test
+
+rubocop
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Emmanuel Simasiku**
+
+- GitHub: [@Mukumbuta](https://github.com/Mukumbuta)
+- Twitter: [@Mukumbuta8](https://twitter.com/Mukumbuta8)
+- LinkedIn: [Mukumbuta](https://linkedin.com/in/mukumbuta)
+
+👤 **Riyana Abdi**
+
+- GitHub: [@githubhandle](https://github.com/ibtisam34)
+- Twitter: [@twitterhandle](https://twitter.com/Queenjin2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/falis-abdikani/)
```

Comment:
Great job @Mukumbuta 🤝

- [ ] You may also remove this second author from your readme file since it is only one author who contributed to this pull request.